tutorials/data_generation_ppi.ipynb

```diff
@@ -139,7 +141,10 @@
     "\tbas = csv_data_indexed.measurement_value.values.tolist()\n",
     "\treturn pdb_files, bas\n",
     "\n",
-    "pdb_files, bas = get_pdb_files_and_target_data(data_path)"
+    "pdb_files, bas = get_pdb_files_and_target_data(data_path)\n",
+    "\n",
+    "if limit_data:\n",
+    "\tpdb_files = pdb_files[:10]"
```

Comment:
with 10, the training crashes because the test set is too small or sth. Ok at 15 though
```suggestion
    "\tpdb_files = pdb_files[:15]"
```